### ISSUE TITLE
Ternary causes "missing whitespace" error when "whitespace:check-operator" enabled

### ISSUE THREAD

**Issue by tony19** [palantir/tslint#1372]
### Bug Report
- **TSLint version**: `3.13.0`
- **TypeScript version**: `1.8.10`
- **Running TSLint via**: `gulp-tslint`
#### TypeScript code being linted

The error occurs for the following cases:

``` ts
return 'xa-input' + (invalidInput ? ' input-error': ''); // target code
return 'xa-input' + ( invalidInput ? ' input-error': '' ) ; // attempt to fix (failed)
return 'xa-input' + ( invalidInput ? ' input-error': '' ); // attempt to fix (failed)
```

with `tslint.json` configuration:

``` json
{
  "extends": "tslint:latest",
  "rules": {
    "quotemark": false,
    "no-console": false,
    "no-reference": false,
    "variable-name": {
      "check-format": "allow-leading-underscore"
    }
  }
}
```
#### Actual behavior

```
[10:12:54] [gulp-tslint] error (whitespace) path/to/foo.ts[42, 56]: missing whitespace
```

When `whitespace:check-operator` is enabled, this error occurs. This forces me to write slightly more verbose code (increases LOC by 1, but still!) to workaround the issue.
#### Expected behavior

No error when whitespace exists around operators in a ternary.

**Comment by andy-hanson** [comment]
`invalidInput ? ' input-error': ''` should be `invalidInput ? ' input-error' : ''`.

**Comment by adidahiya** [comment]
looks like the rule is working as intended.
